src/main/index.ts

```
@@ -196,26 +201,53 @@ function readDirectoryRecursive(dirPath: string): any[] {
     const fullPath = join(dirPath, name)
     const stats = statSync(fullPath)
     const isDirectory = stats.isDirectory()
+    const isFile = stats.isFile()
 
-    return {
+    const node: any = {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)